virttest/utils_disk.py

```diff
@@ -747,7 +833,9 @@ def configure_empty_linux_disk(session, did, size, start="0M", n_partitions=1,
                                        str(start) + size[-1], part_type, timeout)
         start += partition_size
         post_partition = get_linux_disks(session, partition=True).keys()
+        print ("post_partition is %s" % post_partition)
```

Comment:
why not use logging.info

Reply:
oo, it's debug info, will delete them, thanks